core/templates/domain/exploration/WrittenTranslationObjectFactory.ts

```diff
@@ -20,6 +20,10 @@
 import { downgradeInjectable } from '@angular/upgrade/static';
 import { Injectable } from '@angular/core';
 
+export type DataFormat =
+  'html' | 'unicode' | 'set_of_normalized_string' | 'set_of_unicode_string'
+  | 'invalid';
+
```

Comment:
This is same as type `DataFormatToDefaultValuesKey` defined below, can we use that instead ?

Reply:
Is it okay if I add `DataFormatToDefaultValuesKey | 'invalid'` where ever the new `DataFormat` was being used (3 places in this file and 1 time in `WrittenTranslationsObjectFactory.ts`)? Or should I make a new type as DataFormatToDefaultValuesKey | 'invalid'?